src/frameworks/angular/index.ts

```diff
@@ -31,10 +31,14 @@ export async function discover(dir: string): Promise<Discovery | undefined> {
   return { mayWantBackend: !!serverTarget, publicDirectory: join(dir, "src", "assets") };
 }
 
-export async function init(setup: any) {
-  execSync(`npx --yes -p @angular/cli@latest ng new ${setup.hosting.source} --skip-git`, {
-    stdio: "inherit",
-  });
+export async function init(setup: any, config: any) {
```

Comment:
it would be good to have a type for `setup` and `config`

Reply:
SG. I'll take in a follow on as we rework init to be testable.